core/domain/exp_services_test.py

```diff
@@ -1062,6 +1084,305 @@ def test_save_and_retrieve_exploration_summary(self):
         self.assertEqual(retrieved_exp_summary.category, 'A new category')
         self.assertEqual(retrieved_exp_summary.contributor_ids, [self.owner_id])
 
+    def test_apply_change_list(self):
+        self.save_new_linear_exp_with_state_names_and_interactions(
+            self.EXP_0_ID, self.owner_id, ['State 1', 'State 2'],
+            ['TextInput'], category='Algebra',
+            correctness_feedback_enabled=True)
+
+        recorded_voiceovers_dict = {
+            'voiceovers_mapping': {
+                'content': {
+                        'en': {
+                            'filename': 'filename3.mp3',
+                            'file_size_bytes': 3000,
+                            'needs_update': False,
+                            'duration_secs': 42.43
```

Comment:
This block is indented one level too far

Reply:
fixed